.rubocop.yml

```diff
@@ -51,6 +51,17 @@ Lint/HandleExceptions:
   StyleGuide: "#dont-hide-exceptions"
   Enabled: true
 
+# Supports --auto-correct
+Lint/InheritException:
+  Description: Avoid inheriting from the `Exception` class.
+  Enabled: true
+  EnforcedStyle: runtime_error
+  SupportedStyles:
+  - runtime_error
+  - standard_error
+  Exclude:
+  - truffleruby/src/main/ruby/core/exception.rb
```

Comment:
It's the default. (The point is to stick with default when we can.) Exception children are rare, We've implemented most of Ruby and MRI is not adding new children of Exception often either, therefore we'll run into this very rarely.